<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentActionsWindow</class>
 <widget class="QDialog" name="StudentActionsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-image: url(:/bleu/bleu.jpg);</string>
  </property>
  <widget class="QLineEdit" name="lineEditTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
background-image: url(:/white/white.jpg);</string>
   </property>
   <property name="placeholderText">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxType">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Type</string>
   </property>
   <item>
    <property name="text">
     <string>Article</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Book</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DigitalBook</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Thesis</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButtonBorrow">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/white/white.jpg);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
   </property>
   <property name="text">
    <string>Borrow</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReturn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/white/white.jpg);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/white/white.jpg);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
   </property>
   <property name="text">
    <string>See events</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEvent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/white/white.jpg);
color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/bleu/bleu.jpg);
color: rgb(140, 206, 241);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
